From 4708a509aa9d65ae93e9824e42ddbc6e8d42d90c Mon Sep 17 00:00:00 2001
From: Michel Dänzer <michel@daenzer.net>
Date: Sat, 27 Aug 2011 18:04:44 +0000
Subject: Fix PowerPC checks for __NR_perf_counter_open.

__ppc__ isn't defined here on Debian powerpc. Grepping through the headers
installed in /usr/include, there are a few references to __ppc__ and
__ppc64__, but I suspect they're for other OSs.

Signed-off-by: Michel Dänzer <michel@daenzer.net>
---
diff --git a/collector.c b/collector.c
index b28964f..fe16967 100644
--- a/collector.c
+++ b/collector.c
@@ -175,7 +175,7 @@ sysprof_perf_counter_open (struct perf_counter_attr *attr,
 #define __NR_perf_counter_open 337
 #elif defined(__hppa__)
 #define __NR_perf_counter_open 318
-#elif defined(__ppc__) || defined(__ppc64__)
+#elif defined(__powerpc__) || defined(__powerpc64__)
 #define __NR_perf_counter_open 319
 #elif defined(__s390__)
 #define __NR_perf_counter_open 331
--
cgit v0.9.0.2
